lib/builderator/interface/packer.rb

```diff
@@ -138,12 +138,20 @@ def _chef_provisioner_base
           :environments_path => Config.local.environment_path,
           :chef_environment => Config.profile.current.chef.environment,
           :json => Config.profile.current.chef.node_attrs,
+          :staging_directory => Config.chef.staging_directory,
+          :execute_command => _chef_execute_command(true),
+          :install_command => _chef_install_command(true)
```

Comment:
How so?  Won't the _docker function merge its prevent_sudo and install_command on top of the base?  Maybe I could change the docker func to have execute_command(false)?